### ISSUE TITLE
--print-deps-only output is hard to parse

### ISSUE THREAD

**Issue by encukou** [fedora-python/tox-current-env#13]
It seems that `--print-deps-only` output can be parsed with:
```python
lines = tox_output.splitlines()
summary = lines.index(' summary '.center(80, '_'))  # or a regex?
actual_deps = lines[:summary]
```
It would be better to have `--print-deps-to-file=FILENAME` option.

**Comment by hroncok** [comment]
or to find out how to get rid of the summary line.

**Comment by encukou** [comment]
and any other possible lines tox might print to stdout

**Comment by hroncok** [comment]
Printing to file is most likely the safest option.

**Comment by hroncok** [comment]
If the file exists, should it:

 - append
 - override
 - refuse to continue

?

**Comment by hroncok** [comment]
I can work on this once https://github.com/fedora-python/tox-current-env/pull/10 is merged (I want to avoid conflicts).
